File: Php/integration/Message/EmailMessageSender.php
namespace integration\Message{

	class EmailMessageSender implements MessageSender{

		public function send($to, $subject, $body){
			$headers = array();
			$headers[] = "From: PokerBunch.com <noreply@pokerbunch.com>";
			$headers[] = "Content-Type: text/plain; charset=\"iso-8859-1\"";
			$headers[] = "X-Mailer: PHP/" . phpversion();
			return mail($to, $subject, $body, implode("\r\n", $headers));
		}

	}

}

File: Infrastructure/Services/EmailMessageSender.cs
using System.Net.Mail;
using Application.Services.Interfaces;

namespace Infrastructure.Services{

	public class MessageSender : IMessageSender{

		public void Send(string to, string subject, string body)
        {
            var message = new MailMessage
                {
                    From = new MailAddress("PokerBunch.com <noreply@pokerbunch.com>"),
                    Subject = subject,
                    Body = body
                };
		    message.To.Add(new MailAddress(to));
 
            var client = new SmtpClient();
            client.Send(message);
		}

	}

}